### ISSUE TITLE
Feature request: Open URLs in background

### ISSUE THREAD

**Issue by mbrgm** [relikd/baRSS#14]
## Motivation/Use case

One might want to scan all new/unread feed items and choose to open several of them in the background. I.e. the menu should not close after opening an item. The option to do so could be exposed as a user preference.

**Comment by relikd** [comment]
Yeah, I wanted the same feature once or twice already. I am not sure if I can though. The "standard library" (so to speak) has a defined behavior for menu item. I only get a callback that the item was clicked and cannot abort the close. But I'll look into it as soon as possible.

Do you know any other app which has that feature? (clicking a menu item and the menu stays open. [of course, a native macOS menu and no webview or some])

**Comment by KarunaXY** [comment]
Or it could be the other way, i.e., One might want to mark several uninterested items as read in the menu instead of opening to remove them from the feed list. Then one could read interested things leftover one by one.

**Comment by relikd** [comment]
@KarunaXY there is already a feature to hold down option key (and click) to mark an entry as read or unread without opening it. But I assume your comment was stating to use that without the menu closing. I have the same problem here that after a click the callback will close it automatically. So if I can find a solution for the first, there will be also a solution for the mark-as-read feature. ;)

**Comment by KarunaXY** [comment]
@relikd Good to know this option key feature 👍 
> @KarunaXY But I assume your comment was stating to use that without the menu closing. I have the same problem here that after a click the callback will close it automatically. 

Yes, exactly, it would be nice to do it without closing the menu. 😄
